packages/sdk/src/jwt/core/validate.ts

```diff
@@ -63,7 +63,7 @@ export function verifyJWT(jwt: string, expectedAudience: string): VincentJWT {
     const s = signatureBytes.slice(32, 64);
 
     // Process public key
-    let publicKey = decoded.payload.pkpPublicKey;
+    let publicKey = decoded.payload.pkp.publicKey;
```

Comment:
Since this is validating what we just deserialized from a string, we should check that all of the expected Vincent JWT payload properties exist on the JWT and are well formed (like we were doing for `pkpPublicKey` before this change) and not us `as VincentJWT` when we're not actually sure at runtime that it is one.

Reply:
Consumers now would have to use the decoded payload directly. That way we will avoid duplicating references